pom.xml

```diff
@@ -1630,6 +1975,222 @@
                                     </features>
                                 </category>
 
+                                <!--Identity agents category-->
+                                <category>
+                                    <id>org.wso2.carbon.identity.agents.category.feature.category</id>
+                                    <label>Identity Agents</label>
+                                    <description>Includes identity agent features.</description>
+                                    <features>
+                                        <catFeature>
+                                            <id>org.wso2.carbon.identity.agents.category</id>
+                                            <version>${carbon.identity.version.5.0.7}</version>
+                                        </catFeature>
+                                    </features>
+                                </category>
+
+                                <!--Identity authorization category-->
+                                <category>
+                                    <id>org.wso2.carbon.identity.authorization.category.feature.category</id>
+                                    <label>Identity Authorization</label>
+                                    <description>Includes identity authorization features.</description>
+                                    <features>
+                                        <catFeature>
+                                            <id>org.wso2.carbon.identity.authorization.category</id>
+                                            <version>${carbon.identity.version.5.0.7}</version>
+                                        </catFeature>
+                                    </features>
+                                </category>
+
+                                <!--Carbon authenticators category-->
+                                <category>
+                                    <id>org.wso2.carbon.identity.authenticators.category.feature.category</id>
+                                    <label>Carbon Authenticators</label>
```

Comment:
org.wso2.carbon.identity.authorization.category.feature.category and org.wso2.carbon.identity.authenticators.category.feature.category are not in order